recipes/metabat/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: metabat
+  version: 2.12.1
+
+build:
+  number: 0
+  skip: True  # [not linux64 or not py27]
+
+source:
+  url: https://bitbucket.org/berkeleylab/metabat/downloads/metabat-static-binary-linux-x64_v2.12.1.tar.gz
+  sha256: 1fc70cb388feac43392aa87190cd368797cb88dbd0be8b541b8019187436dbc3
+  patches:
+    - metabat.patch
+
+requirements:
+  run:
+    - python
```

Comment:
I think -- but haven't checked that entirely -- that `python` is only a build requirement if you were to build from source.